server/http/restapi/bindings.go

```diff
@@ -8,12 +8,13 @@ import (
 )
 
 func (a *restapi) handleGetBindings(w http.ResponseWriter, req *http.Request, actingUserID string) {
+	query := req.URL.Query()
```

Comment:
why change from req.Form? i thought form is a more generic version, supports query args?

Reply:
@levb It wasn't working for me, the values were not present for some reason.